srcpkgs/libpinyin/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'libpinyin'
+pkgname=libpinyin
+version=2.2.1
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr"
+hostmakedepends="intltool libtool pkg-config autoconf-archive autoconf automake wget"
+makedepends="db-devel libdb-cxx gnome-common glib-devel glib gettext-devel"
```

Comment:
remove glib dependency, replace `glib-devel` with `libglib-devel`

Reply:
> On Mon, Dec 31, 2018 at 08:41:22PM -0800, Ben Hsu wrote: pobetiger commented on this pull request. > @@ -0,0 +1,18 @@ +# Template file for 'ibus-libpinyin' +pkgname=ibus-libpinyin +version=1.10.92 +revision=1 +build_style=gnu-configure +configure_args="--prefix=/usr" Awesome. Will be removed in next commit also. -- You are receiving this because you commented. Reply to this email directly or view it on GitHub: [#6423 (comment)](https://github.com/void-linux/void-packages/pull/6423#discussion_r244628052)
> Please keep all changes related to one package in one single commit, you can make new commits but the final version should have only the 2 commits as they appear right now.

Will do.